documentation/3.4.0/setup/native/samiksha-with-project/README.md

```
@@ -0,0 +1,320 @@
+
+    ```
+    curl -L -o project-service/config.json https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/3.4.0/common-files/generics/configFile.json && \
+    curl -L -o samiksha-service/config.json https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/3.4.0/common-files/generics/configFile.json && \
+    curl -L -o entity-management/src/config.json https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/3.4.0/common-files/generics/configFile.json
+    ```
+
+6.  **Create Databases**
+
+    1. Download `create-databases.sh` Script File:
+
+    ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)